hw/ip/alert_handler/doc/dv_plan/index.md

```diff
@@ -34,6 +34,8 @@ In addition, it instantiates the following interfaces, connects them to the DUT
 * Interrupts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
 * Devmode ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
 
+In chip level testing, alert_handler is able to reuse its testbench environment with an auto-generated chip-level paramter package located at `hw/top_earlgrey/ip/alert_handler/dv/alert_handler_env_pkg__params.sv`
+
```

Comment:
related to PR #3548, will discuss with Sri if we should add top-level related stuff in this file.